packages/flutter_tools/test/integration.shard/expression_evaluation_test.dart

```diff
@@ -189,4 +221,5 @@ void expectValue(ObjRef result, String message) {
 void main() {
   batch1();
   batch2();
+  batch3();
```

Comment:
you have any idea why we batch them like this?

Reply:
Not for certain.. They were added in https://github.com/flutter/flutter/pull/41880/files

I figured maybe it was for some consistency and scoping. Those in each batch use a different `TestProject` and each "batch" has a similar `initProject`, `cleanProject` function. I'm not sure I see a reason why they wouldn't have just been inside `group()`s though. I can try changing them to be that way though (either in this PR or another)?

Reply:
Gonna merge this, but let me know if you think I should try changing these to `group`s in a future CL. Base on other things in the change above, maybe this was intended to provide some sharding, but as far as I can tell nobody is calling these `batch()` functions individually.